app/src/main/java/ar/com/wolox/android/bootstrap/ui/login/LoginFragment.kt

```diff
@@ -6,32 +6,57 @@ import android.view.LayoutInflater
 import android.view.View
 import android.view.ViewGroup
 import androidx.core.net.toUri
+import androidx.fragment.app.Fragment
+import androidx.fragment.app.activityViewModels
 import androidx.fragment.app.viewModels
+import androidx.navigation.fragment.findNavController
 import ar.com.wolox.android.bootstrap.BuildConfig
 import ar.com.wolox.android.bootstrap.R
 import ar.com.wolox.android.bootstrap.databinding.FragmentLoginBinding
-import ar.com.wolox.android.bootstrap.ui.base.BaseFragment
-import ar.com.wolox.android.bootstrap.ui.posts.PostsActivity
+import ar.com.wolox.android.bootstrap.network.util.RequestStatus
+import ar.com.wolox.android.bootstrap.ui.root.RootViewModel
 import ar.com.wolox.android.bootstrap.utils.SnackbarFactory
 import dagger.hilt.android.AndroidEntryPoint
 
 @AndroidEntryPoint
-class LoginFragment : BaseFragment<LoginView, LoginViewModel>(), LoginView {
+class LoginFragment : Fragment() {
```

Comment:
I'm not getting why these basic structures were deleted. `BaseFragment` for example had some open functions that you are using here as private 🤔

Reply:
I think you are right. But I will have to modify some of them. For example the BaseFragment does not need a reference to the View interface anymore. I will do some modifications and push again.